library/aos_version.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
```

Comment:
Two questions:

The two modules are used in 3 roles (`common`, `masters` and `nodes`). If the idea is to avoid symlinks, what would you propose -- create yet another role just to hold the modules and then add it as a dependency of each of the other 3?

Should we consider the existing modules in `/library` and their symlinks a "bug/techdebt" or `modify_yaml.py` and `rpm_q.py` are in their right place?